### PR TITLE
Creator: Scene Explorer with world objects tree

### PR DESCRIPTION
**Pull request by dmitry-yudakov**

### CODE REVIEW

File: packages/app/src/scenes/widgets/pages/CreatorWidget/pages/SceneExplorer/SceneExplorer.styled.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+export const Item = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 0 10px;
+  height: 40px;
+  min-width: 250px;
+  transition: all var(--tr-150-ei);
+
+  border-bottom: 0.6px solid #fff;
```
Comment (PAKovalev) [inline packages/app/src/scenes/widgets/pages/CreatorWidget/pages/SceneExplorer/SceneExplorer.styled.ts]: 1px + color const )

File: packages/app/src/scenes/widgets/pages/CreatorWidget/pages/SceneExplorer/SceneExplorer.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            {hasChildren && isExpanded && <>{renderChildren(child, depth + 1)}</>}
+          </Fragment>
+        );
+      });
+  };
+
+  return (
+    <styled.Container>
+      <styled.Description>
+        Below is a list of all the 3d assets currently present in your odyssey. From here you can
```
Comment (PAKovalev) [inline packages/app/src/scenes/widgets/pages/CreatorWidget/pages/SceneExplorer/SceneExplorer.tsx]: i18n ))

File: packages/app/src/stores/UniverseStore/models/World3dStore/World3dStore.ts [hunk 3]
```diff
@@ -168,6 +178,26 @@ const World3dStore = types
       // UnityService.colorPickedPreview(objectId, colorHex);
     }
   }))
+  .actions((self) => ({
+    openDeleteObjectDialog(objectId: string) {
+      const {
```
Comment (PAKovalev) [inline packages/app/src/stores/UniverseStore/models/World3dStore/World3dStore.ts:183]: Mixing stores. It is not good. Make these calls in the component.

**Review by PAKovalev — APPROVED** [review]

File: packages/app/src/stores/UniverseStore/models/World3dStore/World3dStore.ts (hunk 3 above)
Comment (dmitry-yudakov) [reply packages/app/src/stores/UniverseStore/models/World3dStore/World3dStore.ts:183]: As we agreed, leaving it like this for now - this whole thing needs a bigger refactoring.

**Pull request merged**